README.md

```diff
@@ -131,8 +142,13 @@ Let's say the Posts in the previous question are available when you visit `http:
   3. If Step 1 is unsuccessful, print an error message to the console
 
 ```js
-// Your answer goes here...
-```
+$.getJSON ("app.js", function (json){
```

Comment:
You wouldn't make an AJAX call to a `.js` file. Instead use the base URL provided in the question (plus whatever else needs to be added to the URL).